components/slack/sources/common/base.mjs

```diff
@@ -56,43 +112,52 @@ export default {
           });
           return info.team.name;
         } catch (err) {
-          console.log("Error getting team name, probably need to re-connect the account at pipedream.com/apps", err);
+          console.log(
+            "Error getting team name, probably need to re-connect the account at pipedream.com/apps",
+            err,
+          );
           return id;
         }
       });
     },
     async getLastMessage({
       channel, event_ts,
     }) {
-      return this.maybeCached(`lastMessage:${channel}:${event_ts}`, async () => {
-        const info = await this.slack.sdk().conversations.history({
-          channel,
-          latest: event_ts,
-          limit: 1,
-          inclusive: true,
-        });
+      return this.maybeCached(
+        `lastMessage:${channel}:${event_ts}`,
+        async () => {
+          const info = await this.slack.sdk().conversations.history({
+            channel,
+            latest: event_ts,
+            limit: 1,
+            inclusive: true,
+          });
 
-        return info;
-      });
+          return info;
+        },
+      );
     },
```

Comment:
Is the method `getLastMessage()` being used anywhere? If not, we can remove it.

Reply:
Haven't touched it, but true, looks like it's not being used - I removed it, thanks for the heads up.